CommandLine.c

```diff
@@ -288,12 +290,17 @@ int CommandLine_run(const char* name, int argc, char** argv) {
    else
       setlocale(LC_CTYPE, "");
 
-   CommandLineSettings flags = parseArguments(name, argc, argv);
+   CommandLineStatus status;
+   CommandLineSettings flags;
```

Comment:
Better to not leave things uninitialized here.
```suggestion
   CommandLineStatus status = STATUS_OK;
   CommandLineSettings flags = 0;
```